lib/paypal/access_token.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
You should turn all of these to `# frozen_string_literal: false` in order to appease the linter without changing how they were defined in the original gem, I did an experiment on this branch and one of the request objects was complaining about a frozen string.

After doing that change constants will need to be frozen manually.

Reply:
Done!

Reply:
Those sneaky constants are still missing! ☺️ 

```
Offenses:

lib/paypal/paypal_environment.rb:7:16: C: [Correctable] Style/MutableConstant: Freeze mutable objects assigned to constants.
  SANDBOXAPI = 'https://api.sandbox.paypal.com'
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/paypal/paypal_environment.rb:8:13: C: [Correctable] Style/MutableConstant: Freeze mutable objects assigned to constants.
  LIVEAPI = 'https://api.paypal.com'
            ^^^^^^^^^^^^^^^^^^^^^^^^
lib/paypal/paypal_environment.rb:9:16: C: [Correctable] Style/MutableConstant: Freeze mutable objects assigned to constants.
  SANDBOXWEB = 'https://sandbox.paypal.com'
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/paypal/paypal_environment.rb:10:13: C: [Correctable] Style/MutableConstant: Freeze mutable objects assigned to constants.
  LIVEWEB = 'https://paypal.com'
            ^^^^^^^^^^^^^^^👆^^
```

Reply:
I saw that a little too late... Manually froze them